Lib/fontTools/feaLib/ast.py

```diff
@@ -1861,3 +1874,103 @@ def asFea(self, indent=""):
         fields = ("VertTypoAscender", "VertTypoDescender", "VertTypoLineGap")
         keywords = dict([(x.lower(), x) for x in fields])
         return "{} {};".format(keywords[self.key], self.value)
+
+
+class STATDesignAxis(Statement):
+    """STAT DesignAxis
+
+    Args:
+        tag: a 4 letter axis tag
+        order: an int to specify Axis ordering
+        names: a list of name record tuples, eg (3, 1, 1, "Bold")
```

Comment:
Instead of using a tuple for name record, it might be better to subclass `FeatureNameStatement` and override its `build` method.

Reply:
Actually, there's no reason to subclass FeatureNameStatement. These can just be FeatureNameStatement objects because they behave the same.

Reply:
I was concerned that its `build()` method might not be appropriate for `STAT` table, but if it works then all good.

Reply:
Actually, you're right. The `build()` method for FeatureNameStatement does some stuff we don't want to do with STAT names so I subclassed NameRecord to get what we need.